config/settings.py

```diff
@@ -93,7 +93,7 @@
         "osidb.renderers.OsidbRenderer",
     ],
     "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.LimitOffsetPagination",
-    "PAGE_SIZE": 100,
+    "PAGE_SIZE": 10,
```

Comment:
backwards compat with what  ? This list must be very small.

Reply:
With every client, every client expects the default page size to be 100 and that's a contract that we have with them, same as with anything else really.

e.g. it is not uncommon for clients to do the following requests:

`GET /flaws`
`GET /flaws?offset=100&limit=100`
`GET /flaws?offset=200&limit=100`

furthermore I'm not sure this can be considered a performance optimization, the problem is simply being delayed, clients will have to perform more requests which could be slower, but even if it weren't: you can still bypass the default limit by setting it yourself.

Reply:
but what are 'all the clients' - I only  know of sdengine and bindings using osidb in prod at this time.

Reply:
ok got the answers ... and I see your point, reverting this change for now ;)